wandb/sdk/wandb_login.py

```diff
@@ -55,25 +55,26 @@ def login(
     `verify=True`.
 
     Args:
-        anonymous: (string, optional) Can be "must", "allow", or "never".
+        anonymous: Set to "must", "allow", or "never".
```

Comment:
it can also stay None, when we don't set any value, this makes it seems like it has to be one of these values.